MC-10287: Added documentation of the product catalog and pricing (#141)
cloudops/cloudmc-api-docs: source/includes/administration/_products.md

### Product Catalogs

> The product catalogs determines the product configured for a service type and maybe specific connections.

Reviewer: ```suggestion
The product catalogs determine the product configured for a service type and optionally for specific connections.
```

Author: f428ed5b76ae49b8bf4e644a9c5cd49c51399110

      "serviceType": "cloudca",
      "connectionIds": [],
      "name": {
        "en": "fgsfdg",
        "fr": "sfdg"
      },
      "changes": [],
      "description": {
        "en": "sdfg",
        "fr": "sdg"
      },
      "id": "5ba5df7c-1e60-4d99-869d-dd3d97826b2e",
      "categories": [
        {
          "name": {
            "en": "sdfg",
            "fr": "sdfg"
          },
          "id": "fcd8908d-e1a0-41f2-9c38-fed1d1f77f18"
        }
      ],
      "products": [
        {
          "metricType": "GAUGE",
          "unit": {
            "unit": "HOUR",
            "name": {}
          },
          "period": "HOUR",
          "name": {
            "en": "vCPU",
            "fr": "vCPU"
          },
          "transformer": {
            "type": "PROPORTIONAL_TO_TIME"
          },
          "id": "342f0d9b-3c3c-456d-b853-31d713b79e72",
          "attribute": "RAWUSAGE",
          "source": "1",
          "filters": [],
          "sku": "vCPU",
          "categoryId": "fcd8908d-e1a0-41f2-9c38-fed1d1f77f18"
        }
      ]
    }
  ]
}
```

Attributes | &nbsp;
---- | -----------
`id`<br/>*UUID* | The UUID of the product catalog.

> `name`<br/>*Object* | The name object in each language for the product catalog (en, fr or es).

Reviewer: ```suggestion
`name`<br/>*Object* | The name object in each language for the product catalog (en, fr, or es).
```
technically correct without the comma, i just think its more explicitly clear

Author: f428ed5b76ae49b8bf4e644a9c5cd49c51399110

`description`<br/>*string* | The description object in each language for the product catalog (en, fr or es).
`mode`<br/>*string* | Identify the mode if it is for all service type or specific connections. Possible values: ALL_CONNECTIONS_OF_TYPE, SPECIFIC_CONNECTIONS.

> `serviceType`<br/>*Object* | The service connection type to which is bound the product catalog.

Reviewer: ```suggestion
`serviceType`<br/>*Object* | The service connection type the product catalog is bound to.
```

Author: f428ed5b76ae49b8bf4e644a9c5cd49c51399110

> `connectionIds`<br/>*Array[UUID]* | Array of the UUID service connection to which the catalog is bound to.

Reviewer: ```suggestion
`connectionIds`<br/>*Array[UUID]* | Array of the UUID service connection the catalog is bound to.
```

Author: f428ed5b76ae49b8bf4e644a9c5cd49c51399110

`changes`<br/>*Array[Object]* | Array of all changes done on the product catalog.
`categories`<br/>*Array[Object]* | The list of product categories object.

> `categories.id`<br/>*Array[Object]* | The list of product categories object.

Reviewer: ```suggestion
`categories.id`<br/>*Array[Object]* | The id of product category object.
```

Author: f428ed5b76ae49b8bf4e644a9c5cd49c51399110
